website/src/pages/platform.tsx

```diff
@@ -70,7 +70,7 @@ const PlatformPage: FC = () => {
           <ContentContainer>
             <SectionTitle>Strawberry Shake</SectionTitle>
             <p>
-              Strawberry Shake is our client tool to generates custom .Net
+              Strawberry Shake is a command line tool that generates custom .Net
```

Comment:
This would be incorrect since it's not a command-line tool.

Reply:
Okay, I just seemed to remember using the command line when testing it some month ago, but I'm probably mistaken. Removed the mention.

Reply:
The command line only initializes the config.